app/views/admin/alert_types/school_alert_type_exceptions/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_tag create_multiple_admin_alert_type_school_alert_type_exceptions_path(@alert_type), method: :put do %>
+  <% SchoolGroup.all.order(name: :asc).each do |school_group| %>
```

Comment:
Not keen on all the direcy class access in the view. Can we push some of it to the controller?

Reply:
Sorted now